### PR TITLE
Brings back Daniel and Disco Inferno, makes them, Meteor and Hyperfractal behind emag

### PR DESCRIPTION
**Pull request by CoffeeDragon16**
## About The Pull Request

This does a few things. It makes the Disco Inferno shuttle, and the Daniel shuttle purchasable again. The two, including the Meteor and Gigafractal shuttles are all now 50,000Cr to buy, and require the communications console to be emagged to purchase.

## Why It's Good For The Game

I feel as if this is a reasonable method of taking on grief shuttles. There is clearly a spot for them, although their implementation has been shaky. Shuttles have been taken out of rotation simply because some would frequently grief using them, and I feel like this helps. This essentially locks grief shuttles behind antagonist interference. The price change will also make the Hyperfractal see more use than before, and keep how often these grief shuttles show up low but reasonable with effort, proportional to their impact on the evacuation. To me this seems like a reasonable compromise between those who wish to see the funny kills you shuttle, and making it more of a direct influence by antagonists and not simply griefers.

## Changelog
:cl:
add: Disco Inferno and Daniel Shuttles are now purchaseable again.
tweak: Meteor Shuttle and Hyperfractal shuttles all require the comms console to be emagged to buy, and are 50k credits.
/:cl:

### CODE REVIEW

**Comment by CoffeeDragon16** [comment]
aaaaa git hub hurts me I need to fix it I did it on the wrong branch aaaaa

**Comment by Jared-Fogle** [comment]
You coded this off of another PR branch instead of master.

**Comment by Barhandar** [comment]
You need a rebase to fix the commit log, also, fun fact: any purchasable shuttle can be forced on by Shuttle Crash event.

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -432,6 +432,7 @@
 		authenticated = 2
 	to_chat(user, "<span class='danger'>You scramble the communication routing circuits!</span>")
 	playsound(src, 'sound/machines/terminal_alert.ogg', 50, FALSE)
+	SSshuttle.shuttle_purchase_requirements_met |= SHUTTLE_UNLOCK_DISCOINFERNO
```
Comment (Jared-Fogle) [inline code/game/machinery/computer/communications.dm:435]: This isn't the way to do this. Not only is this only for the inferno, but it also doesn't include the other shuttles.

The correct way to do this is to have a flag on the alternate shuttle datum that marks whether it is emag only or not.

From there, only show those in the shuttle menu when the console is emagged (check how it checks for whether to send messages to the syndicate or CentCom), and only allow them to purchased at this point as well.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by Jared-Fogle** [comment]
@Barhandar 

> fun fact: any purchasable shuttle can be forced on by Shuttle Crash event.

hell yes

File: code/game/machinery/computer/communications.dm (hunk 1 above)
Comment (Jared-Fogle) [reply code/game/machinery/computer/communications.dm]: The force push made this outdated--but it's not.

**Comment by Jared-Fogle** [comment]
You shouldn't have changelog.html in your commits either.

**Comment by CoffeeDragon16** [comment]
I think I need to nuke and re-open the PR I did fuck it good

**Pull request closed without merging**